app/utils/url.js

```diff
@@ -9,3 +9,11 @@ export function isValidUrl(url) {
 export function stripTrailingSlashes(url) {
     return url.replace(/\/+$/, '').trim();
 }
+
+export function removePathUrl(url) {
+    var splitted = url.split('/');
+    while (splitted.length !== 3) {
```

Comment:
this makes me super nervous, what if someone types as the url "asdasd", it will be in an indefinite loop, im I right?